packages/eslint-config/index.js

```diff
@@ -1,3 +1,52 @@
 module.exports = {
-  extends: 'airbnb-base',
-};
+    extends: 'airbnb-base',
+    rules: {
+        'arrow-parens': ['error', 'as-needed'],
+        'comma-dangle': ['error', 'only-multiline'],
+        'eol-last': 'off',
+        /*
+            This is the default value which is defined here to note that there may be exceptions
```

Comment:
I don't think this one matches the default for airbnb (which was `'warn'`) so we can delete the first sentence. I'd also like to add a bit more detailed rationale. How about: "Legacy applications that make heavy use of the IIFE (Immediately invoked function expression) may choose to disable this rule"

Reply:
That was my mistake, good catch.